synapse/lib/cell.py

```diff
@@ -3226,6 +3226,10 @@ def initSslCtx(self, certpath, keypath):
         sslctx.load_cert_chain(certpath, keypath)
         return sslctx
 
+    def _log_web_headers(self):
+        # Return a list of header names that should be logged for each web request
+        return ['user-agent']
```

Comment:
Do we anticipate making this something we would change at runtime? If not; then we could treat this as a class local and we only have to create the headers list.

Reply:
definitely not runtime modifiable. I just thought it would be nice to have the list be flexible.

Reply:
I'll make it a class local instead.